server/shovel.test.ts

```diff
@@ -0,0 +1,41 @@
+import { expect, test, describe } from 'bun:test'
+import db from './src/db'
+
+describe('shovel sanity test', () => {
+  test('db exists', () => {
+    expect(db).toBeDefined()
+  })
+
+  test('events got pulled into tables', async () => {
+    const { rows, rowCount } = await db.query(
+      `
+      WITH tbl AS
+        (SELECT table_schema,
+                TABLE_NAME
+        FROM information_schema.tables
+        WHERE TABLE_NAME not like 'pg_%'
+          AND table_schema in ('public'))
+      SELECT table_schema,
+            TABLE_NAME,
+            (xpath('/row/c/text()', query_to_xml(format('select count(*) as c from %I.%I', table_schema, TABLE_NAME), FALSE, TRUE, '')))[1]::text::int AS rows_n
```

Comment:
this looks wild

Reply:
thank you ai. definitely an unusual query you'd only do in test / dev.